### DISCUSSION TITLE
Where to install gifsicle and exiftool? (Venv, Vladmandic Fork, Windows 11)

### DISCUSSION THREAD

**Discussion by StingPax** [continue-revolution/sd-webui-animatediff#291] [Q&A]
Hello,
i want to optimise my gif output with gifsicle and exiftool.
I use an venv on my windows pc with the vladmandic SD fork.
I got both of them installed in the site-packages folder, but animatediff cant find these there, i put them in the animatediff extention folder and it didnt found them there neither, where should i put them that animatediff can find them?

Thank you very much for your help!

**Comment by continue-revolution** [comment]
@zappityzap

**Comment by zappityzap** [comment]
Each binary will need to placed in a folder that is part of your Windows PATH. You can see your current PATH by opening a command prompt and typing: echo %PATH%

You can test if the binary is in your path by trying to run it manually in a command prompt.

For exiftool, if you get the binary from exiftool.org it will need to be renamed from exiftool(-k).exe to exiftool.exe. If you get the Windows installer from https://oliverbetz.de/pages/Artikel/ExifTool-for-Windows then it may already be renamed, and the installer may place it in your PATH automatically, but I haven't tested it as I don't use Windows.

Please include output from command prompt if you continue to have issues.

**Reply by StingPax** [reply]
Yea, i got both of them working, thank you very much for your help!

**Reply by galahadydl** [reply]
Now I downloaded these two files. Which path should I put them in next?
![animatediff bug](https://github.com/continue-revolution/sd-webui-animatediff/assets/49395381/a1558d80-decf-420d-997b-278af38f8855)
